CHANGES.rst

```diff
@@ -4,78 +4,87 @@
 general
 -------
 
-- Made style changes due to the new 5.0.3 version of flake8, which noted many
+- Made code style changes due to the new 5.0.3 version of flake8, which noted many
   missing white spaces after keywords. [#6958]
 
 ami_analyze
 -----------
 
-- Revert Fourier Transform code to avoid using Poppy which was recently updated
-  to use a different sign convention.[#6967]
+- Revert Fourier Transform code to original standalone module rather than importing
+  from the Poppy package, which was recently updated to use a different sign convention.
+  [#6967]
 
 assign_wcs
 ----------
 
 - Added convenience function ``update_fits_wcsinfo()`` to ``assign_wcs.util``
   module to allow easy updating of FITS WCS stored in ``datamodel.meta.wcsinfo``
-  from data model's GWCS. [#6935]
+  from datamodel's GWCS. [#6935]
```

Comment:
```suggestion
  from ``datamodel``'s GWCS. [#6935]
```

Reply:
All suggestions implemented in latest commit.